test/Makefile.am

```diff
@@ -14,6 +17,26 @@ noinst_PROGRAMS = gran_overhead printwhileon
 check_PROGRAMS = tst_simple global
 TESTS = tst_simple global
 
+# Test autoprofiling if either HAVE_LIBUNWIND or HAVE_BACKTRACE
+if HAVE_LIBUNWIND
+  AM_LDFLAGS += -rdynamic -lunwind
```

Comment:
You may mean cygprofile_LDFLAGS here instead of AM_LDFLAGS.

AM_LDFLAGS will all these flags for all compiles in this directory. cygprofile_LDFLAGS will just add it for that one program.